test/parallel/test-openssl-ca-options.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+// This test check the usage of --use-bundled-ca and --use-openssl-ca arguments
+// to verify that both are not used at the same time.
+require('../common');
+const assert = require('assert');
+const child_process = require('child_process');
+const result = child_process.spawnSync(process.execPath,
+                                      ['--use-bundled-ca', '--use-openssl-ca']);
+
+assert.strictEqual(result.stderr.toString(),
```

Comment:
You can avoid having to call `toString()` here if you pass an `encoding` (e.g. `utf8`) option to `child_process.spawnSync`.